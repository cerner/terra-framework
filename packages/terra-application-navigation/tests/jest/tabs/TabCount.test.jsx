import React from 'react';
import TabCount from '../../../src/tabs/_TabCount';
import mockIntl from '../mockIntl';

describe('TabCount', () => {
  it('should render default element', () => {
    const shallowComponent = shallow(
      <TabCount.WrappedComponent intl={mockIntl} />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });

  it('should render prop data', () => {
    const shallowComponent = shallow(
      <TabCount.WrappedComponent
        intl={mockIntl}
        value={1}
        isRollup
      />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });
});
