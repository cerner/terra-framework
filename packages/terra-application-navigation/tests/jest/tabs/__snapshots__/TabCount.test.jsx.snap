// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabCount should render default element 1`] = `
<div
  className="count"
/>
`;

exports[`TabCount should render prop data 1`] = `
<div
  className="count is-rollup"
>
  Terra.applicationNavigation.notifications.new
</div>
`;
